### DISCUSSION TITLE
How to differentiate Internal and External Team sites using PnP Search Results Webpart

### DISCUSSION THREAD

**Discussion by monadha** [microsoft-search/pnp-modern-search#1355] [Q&A] [answered]
Hi,
I am currently using the PnP Search Results webpart to display all the Team sites using custom KQL Query.

Now I want to differentiate the Search Results based on Site Sharing Capability and display Internal/External Team sites separately on the search result webpart.

I have tried using "ViewableByExternalUsers" property is the KQL Query template. When ViewableByExternalUsers:false, It should display "Internal sites" and when ViewableByExternalUsers:true, it should display "External sites" BUT it does not display the correct search results.

I am not sure if I am using the correct property to distinguish between Internal/External sites or am I missing something?
Has anyone have worked on this issue or is aware of how to differentiate between Internal/External Team sites in PnP Search Results. Let me know.

Thanks.

**Comment by wobba** [comment, marked as answer]
There is unfortuntately no property in search applied to all items to say if they come from a site of a certain sharing setting. If you had internal/external as part of the Site title or URL you could make it work via convention - but that requires implementing the convention on site creation/setup.
